### DISCUSSION TITLE
Installation help

### DISCUSSION THREAD

**Discussion by smantyni** [veista/nilan#32] [Q&A] [answered]
Hi,

I installed the package from HACS. I can see that the files appeared under custom_components/nilan. I restarted HA, but nothing happened, no new entities to work with, no Nilan HVAC to be found.

I have RS485-USB connection to my Nilan EC9 which I can access through modbus integration. I can read the registers but have trouble writing them.

How to troubleshoot the installation issue?

**Comment by veista** [comment, marked as answer]
Hi. Go to settings->devices and services, click add integration, search nilan. Follow the istallation.

**Reply by smantyni** [reply]
Thank you, this and a restart helped! I can now see the device and 49 entities. Maybe add this to readme for dummies like me?

**Reply by veista** [reply]
Great. It's how integrations work in HA. It reads now in here.

**Comment by EliAngre** [comment]
Hi,
I connected Nilan ec9  rs458-usb converter to Home Assistant Green and installed the integration, but the integration did not automatically start working. How can I connect Nilan to the integration?

**Reply by veista** [reply]
Hi. Go to settings->devices and services, click add integration, search nilan. Follow the istallation.

**Reply by EliAngre** [reply]
I go settings -> devices and services -> add integration -> nilan -> select interface type -> serial -> serial configuration, but what is "path"?

**Reply by veista** [reply]
#47
